staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7662,3 +7678,19 @@ const (
 	// the destination set to the node's IP and port or the pod's IP and port.
 	LoadBalancerIPModeProxy LoadBalancerIPMode = "Proxy"
 )
+
+// ImageVolumeSource represents a image volume resource.
+type ImageVolumeSource struct {
```

Comment:
If I want to use a specific pull secret for this image, how do I specify that?  I'm guessing the "normal" keyring made up of node credentials, SA image pull secrets, and podspec image pull secrets, with each potentially valid credential being used.  But we should be clear.

I don't see a need or value in subdividing further at this moment.

Reply:
I added a comment to clarify that, thank you!